--- minimacy/src/hw.c	2022-12-05 18:51:13.000000000 +0100
+++ minimacy/src/hw.c	2022-12-09 15:55:47.122793826 +0100
@@ -90,6 +90,13 @@
 #if defined ON_UNIX || defined ON_MACOS_CMDLINE || defined ON_RASPBERRY64
 void systemExecDir(char* path, int len)
 {
+	char* p = "minimacy-in-gnu-store";
+	if (strlen(p) < len - 2)
+	{
+		strcpy(path, p);
+		systemCleanDir(path);
+		return;
+	}
 	path[0] = 0;
 }
 #endif
